crates/fluvio-smartengine/src/metrics.rs

```diff
@@ -0,0 +1,131 @@
+#[cfg(feature = "otel-metrics")]
+pub(crate) struct ChainMetrics {
+    context: opentelemetry::Context,
+    invocations: opentelemetry::metrics::Counter<u64>,
+    records: opentelemetry::metrics::Counter<u64>,
+    bytes: opentelemetry::metrics::Counter<u64>,
+}
+
+#[cfg(feature = "otel-metrics")]
+impl ChainMetrics {
+    pub(crate) fn new() -> Self {
+        let meter = opentelemetry::global::meter("smartengine");
+        let context = opentelemetry::Context::new();
+        let invocations = meter
+            .u64_counter("fluvio.smartengine.chain.invocations")
+            .init();
+        let records = meter.u64_counter("fluvio.smartengine.chain.records").init();
+        let bytes = meter.u64_counter("fluvio.smartengine.chain.bytes").init();
```

Comment:
In the conventions, I think they use `io` instead of `bytes` because "bytes" is a unit used in the `unit` field.

Reply:
ok. We will change it later. It won't be used in first version on UI, anyway